flakeModules/pkgs.nix

```diff
@@ -428,6 +436,7 @@ in
             # TODO: Fix the missing meta/version info upstream
             (mkPkg "bech32" caPkgs."bech32-input-output-hk-cardano-node-10-2-1-52b708f")
             (mkPkg "blockperf" caPkgs.blockperf-cardano-foundation-blockperf-main-87f6f67)
+            (mkPkg "blockfrost-platform" caPkgs.default-blockfrost-blockfrost-platform-0-0-2-e06029b)
```

Comment:
If you want to have a description of this package in the devShell menu, you could also append a `meta.description` here like `cardano-wallet` does below.  In any case, if a description is added to the upstream package in the future, it will get picked up here automatically when we bump to it. :+1: